src/index.html

```diff
@@ -5,8 +5,293 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__icon">
+        <img src="images/logo.png" alt="">
+      </a>
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__item">
+            <a href="#apple" class="header__link is-active">APPLE</a>
+          </li>
+          <li class="header__item">
+            <a href="#samsung" class="header__link">SAMSUNG</a>
+          </li>
+          <li class="header__item">
+            <a href="#смартфоны_и_телефоны" class="header__link">СМАРТФОНЫ И ТЕЛЕФОНЫ</a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#ноутбуки_и_компьютеры"
+              class="header__link"
+              data-qa="nav-hover"
+            >НОУТБУКИ И КОМПЬЮТЕРЫ
+            </a>
+          </li>
+          <li class="header__item">
+            <a href="#гаджеты" class="header__link">ГАДЖЕТЫ</a>
+          </li>
+          <li class="header__item">
+            <a href="#планшеты" class="header__link">ПЛАНШЕТЫ</a>
+          </li>
+          <li class="header__item">
+            <a href="#фото" class="header__link">ФОТО</a>
+          </li>
+          <li class="header__item">
+            <a href="#видео" class="header__link">ВИДЕО</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="catalog">
+        <div class="card" data-qa="card">
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="imac"
+          >
+          <h2 class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+          <p class="card__code">Код товара: 195434</p>
+          <div class="card__feedback">
```

Comment:
```suggestion
          <p class="card__code">Код товара: 195434</p>

          <div class="card__feedback">
```
It's better to separate logical elements with a blank line